doc/ref/configuration/master.rst

```diff
@@ -2353,6 +2324,21 @@ strategy between different sources. It accepts 4 values:
 
   Guesses the best strategy based on the "renderer" setting.
 
+``pillar_merge_lists``
+----------------------------------
+
+.. versionadded:: 2015.8.0
+
+Default: ``True``
```

Comment:
@basepi, I did PRs 30602 and 30609 this morning to address this which has already been merged.